### DISCUSSION TITLE
specifying -lang still returns English output

### DISCUSSION THREAD

**Discussion by miguel-lansdorf** [Tufin/oasdiff#295] [Q&A] [answered]
I've tried supplying the -lang parameter for generating the diff with other language codes (nl, de, fr, cn, zh, etc) but the output is always in English as if this flag is ignored. I installed all locales on my system (confirmed by running locale -a) and I am using a local install of oasdiff and not the docker image.

Is there anything that  I am doing incorrectly?

**Comment by reuvenharrison** [comment, marked as answer]
Hi @miguel-lansdorf ,
Currently only English (en) and Russian (ru) are supported.
I'll update the documentation to emphasize this.
If you'd like to add support for other languages I would be happy to help you do it.

Have a good day,
Reuven
